<ui version="4.0" >
 <class>PrintOptionsPage</class>
 <widget class="QWidget" name="PrintOptionsPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Image Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Image Position</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <item>
       <widget class="QFrame" name="mPositionFrame" />
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" >
        <item>
         <widget class="KHBox" name="cmbox" />
        </item>
        <item>
         <widget class="QCheckBox" name="kcfg_PrintAutoRotate" >
          <property name="text" >
           <string>Auto rotate</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="spacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>213</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Scaling</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="margin" >
       <number>9</number>
      </property>
      <item>
       <widget class="QRadioButton" name="mNoScale" >
        <property name="text" >
         <string>&amp;No scaling</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="mScaleToPage" >
        <property name="text" >
         <string>&amp;Fit image to page</string>
        </property>
        <property name="checked" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="margin" >
         <number>0</number>
        </property>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="kcfg_PrintEnlargeSmallerImages" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>Enlarge smaller images</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>240</width>
            <height>21</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="mScaleTo" >
        <property name="text" >
         <string>&amp;Scale to:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="margin" >
         <number>0</number>
        </property>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="kcfg_PrintWidth" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="value" >
           <double>15.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel2" >
          <property name="text" >
           <string comment="Dimension separator, as in: '15 x 10 centimeters'" >x</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="kcfg_PrintHeight" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="value" >
           <double>10.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KComboBox" name="kcfg_PrintUnit" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="currentIndex" >
           <number>1</number>
          </property>
          <item>
           <property name="text" >
            <string>Millimeters</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Centimeters</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Inches</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="kcfg_PrintKeepRatio" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>Keep ratio</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>16</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
     <zorder>mNoScale</zorder>
     <zorder>mScaleToPage</zorder>
     <zorder>mScaleTo</zorder>
     <zorder>kcfg_PrintAutoRotate</zorder>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>493</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
  <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KHBox</class>
   <extends>QFrame</extends>
   <header location="global" >khbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >kcombobox.h</include>
  <include location="local" >knuminput.h</include>
  <include location="local" >knuminput.h</include>
  <include location="local" >kcombobox.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>mScaleTo</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_PrintUnit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mScaleTo</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_PrintKeepRatio</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mScaleToPage</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_PrintEnlargeSmallerImages</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mScaleTo</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_PrintWidth</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mScaleTo</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_PrintHeight</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
